frontend/src/libs/components/card/card.tsx

```diff
@@ -11,6 +11,7 @@ type Properties = {
   imageUrl?: string;
   onClick: () => void;
   isActive?: boolean;
+  isTooltipAvailable?: boolean;
```

Comment:
```suggestion
  hasTooltip?: boolean;
```
Maybe it's better to rename it like this?

Reply:
I wanted to write that, but I saw Liza was saying somewhere, that it's better if variables are written in one style

Reply:
@Liza-Veis said that we should stick to the one style in the app about naming, so maybe the name with the `is` prefix would suit better

Reply:
Okey, agree